policyengine_us/parameters/gov/states/mn/tax/income/deductions/itemized/reduction/alternate/rate.yaml

```diff
@@ -0,0 +1,17 @@
+description: Minnesota reduces the itemized deductions when federal adjusted gross income is above a threshold by applying this fraction to a subset of itemized deductions.
+values:
+  2021-01-01: 0.8
+
+metadata:
+  period: year
+  unit: /1
+  label: Minnesota alternate itemized deduction reduction rate
+  reference:
+  - title: Minnesota Statutes 290.0123 STANDARD DEDUCTION. Subd. 5.Deduction limited. (b)
+    href: https://www.revisor.mn.gov/statutes/cite/290.0123#stat.290.0123.5
```

Comment:
This source refers to the standard not itemized deduction:
>(b) Notwithstanding paragraph (a), for a taxpayer with adjusted gross income over $1,000,000, the standard deduction is reduced by 80 percent of the standard deduction otherwise allowable under this section.